packages/cli/src/commands/config/add.ts

```diff
@@ -25,9 +25,14 @@ export function Add(_: IContext): CommandModule<{}, Args> {
         demandOption: true,
         choices: fs
           .readdirSync(configsPath)
-          .map((name) =>
-            fs.readdirSync(path.join(configsPath, name)).map((type) => `${name}.${type}`)
-          )
+          .map((name) => {
+            const language = Project.load().language;
```

Comment:
this is causing the project to throw an error when running `teams config add --help`, which should be allowed even when not in a project folder.

Reply:
fixed it